schemas/test/en/test_dynamic_answers_list_source.json

```diff
@@ -156,7 +156,7 @@
                                             "type": "Number",
                                             "decimal_places": 2,
                                             "minimum": {
-                                                "value": 1001
+                                                "value": 1000.99
```

Comment:
I think it would be good to have a functional test for this to make sure it's working properly, with correct error message, i.e. you can enter 1000.99 but not 1000.98

Reply:
I've added a test in 👍